### ISSUE TITLE
a function named pending () triggered the 'no-disabled-tests' error

### ISSUE THREAD

**Issue by lsuarez-tiempo-dev** [jest-community/eslint-plugin-jest#149]
I just read that pending is a jasmine global that allows you not run the test. 
effectively disabling it. 

Our code imports a value and then runs its it. 
It appears the parser is confusing it with the jasmine global.
without being to careful with the syntax, This is basically what we're doing:
```js

import { pending } from './myActions' ; 

describe('the pending action creator should return correct object', ()=>{
    it('should work correctly' , ()=>{
      expect(pending()).toStrictEqual({type:'FETCH_ALL_ALBUMS_PENDING'});
   })
})
```

does this rule notice if you import the function before assuming you're using the jasmine global and hence, disabling the test?  
or is it an issue with the code?
